src/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header id="header">
+      <div id="left_logo"></div>
+      <nav id="humburger_menu">
+        <img src="../images/ic_menu.png" />
+      </nav>
+    </header>
+    <section id="headline">
+      <h1 id="me_names">Derick Zihalirwa</h1>
+      <p id="me_descriptions">
+        Hello! I am a software developer! I can help you build a product,
+        feature or website. Take a look of my works. If you like what you see
+        and have a project you need coded, don’t hesitate and contact me.
+      </p>
+      <a href="#" id="start_collab">Start collaboration</a>
+    </section>
```

Comment:
I can see you have put in a lot of effort in implementing the suggestions of the previous reviewers👍👍, but you still haven't kept to the design fully. kindly fix the required changes of the previous reviewer🤗
![reviewers_suggestion](https://user-images.githubusercontent.com/64643995/212389677-8d38d69e-f7bb-4ec1-8d1c-001ff4b2d82c.png)

Kindly implement this suggestion the previous reviewer dropped for you🤗